src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.cc

```diff
@@ -174,7 +174,6 @@ btree_range_pin_t &BtreeLBAManager::get_pin(CachedExtent &e)
       e.cast<LogicalCachedExtent>()->get_pin()).pin;
   } else {
     assert(0 == "impossible");
-    return *static_cast<btree_range_pin_t*>(nullptr);
```

Comment:
How about production builds where the `assert()` above is compiled-out? Maybe `ceph_abort_msg()` instead? It's already marked as `[[noreturn]]`.

Reply:
@rzarzynski thanks, updated and repushed.